package.json

```diff
@@ -179,8 +178,8 @@
     "react-native-randombytes": "^3.6.0",
     "react-native-reanimated": "^2.5.0",
     "react-native-restart-android": "^0.0.7",
-    "react-native-safe-area-context": "^3.3.2",
-    "react-native-screens": "^3.13.1",
+    "react-native-safe-area-context": "^4.4.1",
```

Comment:
do we need to upgrade this package as part of using `@react-navigation/native-stack`? this upgrade introduced a lot of snapshot updates which has bloated this PR and made it harder to review, if it's not a direct dependency then we should refrain from sneaking this in...

Reply:
@kathaypacific I was able to pull out the upgrades to `react-native-safe-area-context` and `react-native-screens` which should clean-up a lot of the snapshot bloat without issues. Downgrading `@react-navigation/native` produces some intermediate errors that are fixed by the upgrade, so I'd rather include that in this PR instead of fixing the intermediate errors that are fixed by the upgrade.